docs/introduction/faq-and-common-issues.md

```diff
@@ -0,0 +1,74 @@
+# FAQ and Common Issues
+
+This section provides only the basic answers to some of the most frequently asked questions.
+For more detailed information about the Shopsys Framework, please see [Shopsys Framework Knowledge Base](../index.md).
+
+## What are the phing targets?
+Every phing target is a task that can be executed simply by php phing <target-name> command.
+See more about phing targets in [Console Commands for Application Management (Phing Targets)](./console-commands-for-application-management-phing-targets.md).
+
+## What are the data fixtures good for?
+Data fixtures are actually demo data available in the Shopys Framework.
+There are two kinds of demo data, the singledomain and the multidomain.
+For the installation of the basic demo data, you need to have just one domain configured, use the phing target `db-fixtures-demo-singledomain`.
```

Comment:
First - this changed sentence is not right. If I have 2 domains, I have to run `db-fixtures-demo-singledomain` too (and then `db-fixtures-demo-multidomain`)
And again - if I just run these targets, I end up with an error, because these targets expect the system in a certain state.

I recommend to leave here just `db-demo` as it works in any condition - first it wipes public scheme.